compiler/rustc_hir/src/hir.rs

```diff
@@ -4224,7 +4224,7 @@ impl fmt::Display for Constness {
     }
 }
 
-/// The actualy safety specified in syntax. We may treat
+/// The actually safety specified in syntax. We may treat
```

Comment:
I think this should be "actual"? That makes more sense than "actually", though I'm not certain.

@oli-obk: git tells me you wrote this comment, can you confirm?

Reply:
done